Intialise.sql

```diff
@@ -0,0 +1,21 @@
+Create Database week2_SeanM;
+
+Use week2_SeanM;
+
+CREATE TABLE Employee (
+	EmployeeID SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT,
+	Name varchar(60),
+	Salary decimal,
+	Bank_Acc_Number char(8),
+	NI_Number char(9),
+	PRIMARY KEY (EmployeeID)	
+);
+
+create Table DeliveryEmployee ( 
+	DeliveryEmployee SMALLINT UNSIGNED NOT NULL AUTO_INCREMENT,
+    EmployeeID SMALLINT UNSIGNED NOT NULL,
+    PRIMARY KEY (DeliveryEmployee,EmployeeID),
```

Comment:
you don't DeliveryEmployee column in here as this prevents you from having data integrity on the EmployeeID column alone due to your use of a composite key. please remove this column and update the primary key